JamfSync/UI/FileSharePasswordView.swift

```diff
@@ -19,33 +21,35 @@ struct FileSharePasswordView: View {
                 .font(.title)
             Text("Enter the password for \(fileShareDp?.selectionName() ?? ""): \(fileShareDp?.address ?? "")")
                 .padding(.bottom)
-
-            HStack {
-                VStack(alignment: .trailing) {
-                    Text("Password:")
-                        .frame(height: 16)
-                }
-                VStack {
-                    HStack {
-                        SecureField(text: $password, prompt: Text("Password")) {
-                            Text("Title")
-                        }
-                        Toggle(isOn: $saveInKeychain) {
-                            Text("Save in Keychain")
-                        }
-                    }
-                    .frame(height: 16)
-                }
+            
+            Form {
+                Section(header: Text("Account:")  // Label
+                    .font(.headline)
+                    .padding(EdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 0))
+                ) {
+                    TextField("", text: $rwUsername, prompt: Text(""))
+                        .disabled(true)
+                       .frame(width: 420, height: 30, alignment: .leading)
```

Comment:
It's best to avoid frames altogether if you can, but if not, then just specify the parts of the frame that you really need (like height maybe).